test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json

```diff
@@ -7,7 +7,7 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.AspNetCore.Server.Kestrel.Https": "1.0.0-*",
     "Microsoft.AspNetCore.Testing": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1-beta1",
```

Comment:
While you are at it, please remove `"portable-dnxcore50+net45+win8+wp8+wpa81"` from `"imports"` array. That is not needed anymore.